packages/swapper/src/swappers/ChainflipSwapper/utils/getQuoteOrRate.ts

```diff
@@ -277,10 +280,7 @@ export const getRateOrQuote = async (
     }
   }
 
-  // A quote is a rate with guaranteed BIP44 params (account number/sender),
-  // so we can return quotes which can be used as rates, but not the other way around
-  // The 'input' determines if this is a rate or a quote based on the accountNumber and receiveAddress fields
-  const ratesOrQuotes: TradeQuote[] = []
+  const ratesOrQuotes = []
```

Comment:
Removed inline type since TS would get confused with the push and think `(TradeQuote | TradeRate)[]` vs. `TradeQuote[] | TradeRate[]` and we'd end up with ugly casts